spec/index.html

```diff
@@ -686,6 +730,87 @@ <h3>Collections</h3>
     </pre>
 
   </section>
+
+  <section id="quoted-triples">
+    <h3>Quoted Triples</h3>
+
+    <p>A <dfn data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</dfn>
+      may be the <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> or
+      <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of an
+      <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>.</p>
+
+    <p>A <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>
+      is represented using the <code><a href="#grammar-production-quotedTriple">quotedTriple</a></code> production
+      with <code><a href="#grammar-production-subject">subject</a></code>,
+      <code><a href="#grammar-production-predicate">predicate</a></code>, and
+      <code><a href="#grammar-production-object">object</a></code>
+      preceded by <code>&lt;&lt;</code> and followed by <code>&gt;&gt;</code>.
+      Note that <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triples</a>
+      may be recursive.
```

Comment:
```suggestion
      may be nested.
```
"recursive" suggests a quoted triple can refer to itself - a cycle.

The definition of quoted triples is recursive (the rules refer back to themselves), the outcome is not.